docs/snapshotting/snapshot-support.md

```diff
@@ -38,7 +36,23 @@ This has the advantage of very fast snapshot loading times, but comes with the c
 of having to keep the guest memory file around for the entire lifetime of the
 resumed microVM.
 
-*Note*: Snapshotting is currently supported only on `x86_64` machines.
+*Note*: Snapshotting is currently supported on Intel and AMD Zen `x86_64` CPUs.
```

Comment:
I would say Intel x86_64 with virtualization support and AMD Zen CPUs

Reply:
I would not add the virtualization support requirement since it is redundant. Firecracker only works with HW virtualization support.

But, I need to make this accurate wrt our testing. That means we officially support Amd Zen2 `x86_64` CPUs while others may also work without issues.

Reply:
done